.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Go
+
+on:
+  push:
+    branches: ['main', '*.*-dev', '*.*.*-dev']
+  pull_request:
+    branches: [main]
+
+env:
+  ORACLE_INSTANT_CLIENT_VERSION: "21.5.0.0.0-1"
+
+jobs:
+  integration-tests:
+    strategy:
+      fail-fast: false
+
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: "1.23.1"
+
+    - name: Build
+      run: |
+        cd yb-voyager
+        go build -v ./...
+
+    # required by godror driver used in the tests
+    - name: Install Oracle Instant Clients
```

Comment:
Q: any specific reason we are not using the installer script here?

Reply:
we can add installer script later if required. also modified the installer script to skip debezium in a separate PR.